AK/Time.cpp

```diff
@@ -38,6 +41,299 @@ unsigned day_of_week(int year, unsigned month, int day)
     return (year + year / 4 - year / 100 + year / 400 + seek_table[month - 1] + day) % 7;
 }
 
+#ifndef KERNEL
+Optional<struct tm> convert_formatted_string_to_timespec(StringView input, StringView format, size_t& string_pos)
+{
+    [[maybe_unused]] size_t format_pos = 0;
```

Comment:
Surely, removing the `[[maybe_unused]]` here won't make it complain that `format_pos` is unused, right?